Cargo.toml

```diff
@@ -98,11 +101,6 @@ sqlx = { version = "0.7", optional = true, default-features = false, features =
 thiserror = "1"
 tokio = { version = "1", default-features = false, optional = true }
 
-# Temporary until https://github.com/georust/geozero/pull/208 is merged and released.
```

Comment:
Just checking that this is resolved

Reply:
Looks like it was released in v0.13.0 (https://github.com/georust/geozero/blob/f6e06e298a63d15daabac7c2a66fc68e9432c5c6/geozero/CHANGELOG.md?plain=1#L13-L15) and we're on that: https://github.com/geoarrow/geoarrow-rs/blob/53ce09d64f76b2609961dc15604d62ff1ed6f921/Cargo.toml#L72

Reply:
🙏